srcpkgs/fah/files/FAHClient/run

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+OPTS=--run-as=_fah
+set -e
+[ -f config ] && . config
+cd /usr/libexec/_fah && exec /usr/bin/FAHClient $OPTS 2>&1
```

Comment:
does it require to be started as root really? better start the service via `chpst -u _fah:_fah` instead?

Reply:
`FAHClient` has its own `--run-as` parameter to give up rights and run under a different user. I think this would interfere witch `chpst -u _fah:_fah`.

Reply:
better save than sorry, don't you think? if it works without `--run-as` and it doesn't behave differently (as in: does not really require root privilege), i prefer to not start it elevated at all.